packages/lib/modules/lbp/pool/usePriceInfo.ts

```diff
@@ -14,19 +22,29 @@ export type LbpPrice = {
   projectTokenPrice: number
 }
 
+export type HourlyDataPoint = {
+  timestamp: number
+  tvl: number
+  fees: number
+  volume: number
+}
+
 export function usePriceInfo(chain: GqlChain, poolId: Address) {
   const apiResult = useQuery(GetLbpPriceInfoDocument, {
     variables: {
       poolId: poolId.toLowerCase(),
       chain,
+      dataPoints: 5000,
```

Comment:
Aren't those too many? if the recommended length of an LBP is 5 days and we want hourly snapshots that would be 120, even for a month sale it would be around 720.